{
  try {
    namingContext.rebind(new CompositeName("subcontext/test"),new Object());
    fail("Should have thrown and NameNotFoundException");
  }
 catch (  NameNotFoundException expected) {
  }
}
